__tests__/atlas.test.ts

```diff
@@ -482,7 +482,7 @@ describe('all reports with pull request', () => {
     expect(spyOnError).toHaveBeenCalledTimes(0)
     expect(spyOnNotice).toHaveBeenNthCalledWith(
       1,
-      'Adding a unique index "uniq_name" on table "users" might fail in case column "name" contains duplicate entries',
+      'Adding a unique index "uniq_name" on table "users" might fail in case column "name" contains duplicate entries (MF101)\n\nDetails: https://atlasgo.io/lint/analyzers#MF101',
```

Comment:
Do the comments support colors like "yellow" for issues?

Reply:
yes, see that this is an assertion that the `notice` method was invoked. (`error` and `info` also supported)

Reply:
So it's not critical for now since we are still building it, but issues in nebula are yellow so if this should be an issue in Nebula it should be yellow also here

Reply:
we cant control the color unfortunately, but notice should map to "yellow", agree